src/components/Editor.tsx

```diff
@@ -69,7 +83,7 @@ export const Editor = ({
     }
 
     return getLocalStorageKey(
-      !!templateMetadata.isThemeMode,
+      templateMetadata.isThemeMode,
```

Comment:
Is this needed if it's specific for the veConfig and you now have buildThemeLocalStorageKey?